pkg/components/helpers/cache/expirable_lru_test.go

```diff
@@ -0,0 +1,178 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package cache
+
+import (
+	"testing"
+	"testing/synctest"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const testTTL = 5 * time.Minute
+
+func TestExpirableLRU_PutGetRemove(t *testing.T) {
+	lru := NewExpirableLRU[string, string](testTTL)
+
+	lru.Put("foo", "bar")
+	v, ok := lru.Get("foo")
+	assert.True(t, ok)
+	assert.Equal(t, "bar", v)
+	assert.Equal(t, 1, lru.Len())
+
+	_, ok = lru.Get("baz")
+	assert.False(t, ok)
```

Comment:
nit: we should ensure the empty value is also returned

```suggestion
	v, ok = lru.Get("baz")
	assert.False(t, ok)
	assert.Empty(t, v)
```